kivy/config.py

```diff
@@ -315,6 +308,13 @@
 from kivy.compat import PY2, string_types
 from weakref import ref
 
+__all__ = ('Config', 'ConfigParser')
+
+try:
+    from ConfigParser import ConfigParser as PythonConfigParser
```

Comment:
not sure about `ConfigParser`, but `Config` has to be on top, so this is probably bad too